src/server.js

```diff
@@ -1,10 +1,62 @@
 import app from './app';
 import prop from './config/config';
+import socket from 'socket.io';
 
 const currentConfig = prop[process.env.NODE_ENV];
 const { port } = currentConfig;
 const server = app.listen(port, () =>
   console.log(`App listening on ${port}!....`)
 );
 
+const io = socket(server, {
+  cors: {
+    origin: process.env.SOCKET_FRONTED_URL,
+    method: ['GET', 'POST'],
+  },
+});
+
+io.on('connection', (client) => {
+  console.log('Client connected successfully! 🔥 ' + client.id);
```

Comment:
we do not really need these console statements

Reply:
I have now commented that line of code.